2015-04-21 08:58:14,496  INFO [net.jini.lookup.ServiceDiscoveryManager] - Exception occurred during query, discarding reggie [com.sun.jini.reggie.RegistrarProxy[registrar=bb59ab03-2e81-4720-8995-a6d218ac57e5 DynamicSmartStub [ImplObjClass: com.sun.jini.reggie.GigaRegistrar, ConnectionURL: NIO://kafka.local:54556/pid[40994]/1429517374140307232_2_-706347898903840095_details[class com.sun.jini.reggie.GigaRegistrar], MaxConnPool: 1024 ]]]; Caused by: java.rmi.ConnectException: Connect Failed to [NIO://kafka.local:54556/pid[40994]/1429517374140307232_2_-706347898903840095_details[class com.sun.jini.reggie.GigaRegistrar]]; nested exception is: 
	java.net.SocketTimeoutException
2015-04-21 08:58:34,164  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery; Caused by: java.lang.IllegalStateException: discovery terminated
	at net.jini.discovery.LookupDiscovery.discard(LookupDiscovery.java:2051)
	at net.jini.discovery.LookupDiscovery.getActualGroups(LookupDiscovery.java:2968)
	at net.jini.discovery.LookupDiscovery.maybeSendEvent(LookupDiscovery.java:2784)
	at net.jini.discovery.LookupDiscovery.access$3900(LookupDiscovery.java:696)
	at net.jini.discovery.LookupDiscovery$UnicastDiscoveryTask.run(LookupDiscovery.java:1797)
	at com.sun.jini.thread.TaskManager$TaskThread.run(TaskManager.java:408)

2015-04-21 08:59:14,830  INFO [net.jini.lookup.ServiceDiscoveryManager] - Exception occurred while attempting to register with the lookup service event mechanism; Caused by: java.rmi.ConnectException: Connect Failed to [NIO://kafka.local:54556/pid[40994]/1429517374140307232_2_-706347898903840095_details[class com.sun.jini.reggie.GigaRegistrar]]; nested exception is: 
	java.net.SocketTimeoutException
2015-04-21 09:01:41,415  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 09:05:41,472  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 09:09:40,117  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 09:13:40,356  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 09:17:40,343  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 09:21:40,113  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 09:25:39,886  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 09:29:39,511  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 09:33:39,678  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 09:37:39,503  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 09:41:19,798  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.SocketException: Network is down
2015-04-21 09:46:35,264  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 09:50:34,958  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 09:54:35,162  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 09:58:35,058  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 10:02:35,246  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 10:06:35,312  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 10:10:35,621  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 10:14:35,903  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 10:18:36,065  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 10:22:35,234  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 10:26:34,865  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 10:30:36,441  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 10:34:35,563  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 10:38:35,077  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 10:42:35,183  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
2015-04-21 10:46:34,896  INFO [net.jini.discovery.LookupDiscovery] - exception occurred during unicast discovery to kafka.local:4174 with constraints InvocationConstraints[reqs: {}, prefs: {}]; Caused by: java.net.ConnectException: Operation timed out
